src/mapblock.cpp

```diff
@@ -558,8 +558,9 @@ void MapBlock::deSerializeNetworkSpecific(std::istream &is)
 {
 	try {
 		const u8 version = readU8(is);
-		//if(version != 1)
-		//	throw SerializationError("unsupported MapBlock version");
+		if (version != 1) {
```

Comment:
noop this should not be enabled

Reply:
? 'version' has to be used, as it was before bba4563d89b6708d75a4053c69873dff0d747538
What is your code suggestion? Maybe there's another way.

Reply:
Oh, just:
`readU8(is);`
?

Reply:
Ok will do.

Reply:
Done.